pkg/goods/static/99-embedded-cluster.conf

```diff
@@ -0,0 +1,9 @@
+# this entry enables ip forwarding. this feature is necessary as embedded
+# cluster creates virtual network interfaces and need the traffic among them to
+# be forwarded.
+net.ipv4.ip_forward = 1
+
+# arp filter and ignore need to be disabled otherwise we can't have arp
+# resolving across the calico network interfaces.
+net.ipv4.conf.default.arp_filter = 0
+net.ipv4.conf.default.arp_ignore = 0
```

Comment:
We need to do the same for `net.ipv4.conf.all.*` too:
- https://www.kernel.org/doc/html/latest/networking/ip-sysctl.html

> The max value from conf/{all,interface}/arp_ignore is used when ARP request is received on the {interface}

> arp_filter for the interface will be enabled if at least one of conf/{all,interface}/arp_filter is set to TRUE, it will be disabled otherwise

Reply:
Makes sense to me. Addressed.